### DISCUSSION TITLE
vom Wallet zum Füllfederhalter - SSS: Self-Sovereign Signatures?

### DISCUSSION THREAD

**Discussion by mdoujak** [e-id-admin/governance-sounding-board#31] [Q&A]
Ich beschäftige mich aktuell mit verschiedenen Use Case Szenarien für die Verwendung von SSI. 

Dabei fällt mir immer wieder auf, dass der Holder auf das Präsentieren von VPs beschränkt ist. Ich frage mich darum, ob jemand einen Vorschlag, Draft oder Standard kennt, welcher die willensbekundende Unterschrift mittels Self-Sovereign Identities spezifiziert.

Folgende Use Case Szenarien wären hier spannend:
- Einseitige Dokumentation einer Entscheidung (Beispiel: ich bin Organspender)
- Gegenseitige Dokumentation einer Entscheidung (Beispiel: X verkauft Y Objekt O zum Preis von ?? CHF)
- Vertrag mit mehreren Parteien (Beispiel Unterschrift zu zweien, Rollen VCs, ...)

Folgende Problemstellungen müssten in so einem Papier diskutiert werden
- Zeitstempel
  * Beweis dass der Vertrag zu diesem Zeitpunkt vom Signierenden unterschrieben wurde?
  * Beweis dass der Zeitstempel zu diesem Vertrag gehört?
- Welche VCs werden mit dem Vertrag gebündelt? Was ist nötig, damit der Vertrag verifiziert werden kann?
  - Identifikation der Person
  - Zugehörigkeit zu einer Organisation zum Zeitpunkt der Unterschrift
  - Unterschriftenberechtigung
- Welche Eigenschaften soll der Vertragsinhalt haben
  - Unveränderliche Darstellung des Inhalts für alle Unterschreibenden (Beispiel: Nachladen aus dem Internet).
  - Lesbare Darstellung des Inhalts (Beispiel: keine weisse Schrift auf weissem Grund)

**Comment by schischuMM** [comment]
https://didsign.io/
